client/landing/stepper/declarative-flow/internals/steps-repository/ecommerce-segmentation-survey/questions.ts

```diff
@@ -0,0 +1,66 @@
+// import { translate } from 'i18n-calypso';
+const translate = ( text: string ) => text; // Stubbed for now
+import { Question, QuestionType } from 'calypso/components/paginated-survey/types';
+
+export const questions: Question[] = [
+	{
+		headerText: translate( 'What is your favorite color?' ),
```

Comment:
This is just a mock, but I think the questions should be moved to server-side (cc @JuanLucha).